todo/src/reducers/index.js

```diff
@@ -0,0 +1,20 @@
+import { ADD_TODO, UPDATE_TODO, REMOVE_TODO } from '../actions';
+
+const initialState = [{ text: 'Clean Garage', completed: false, id: 0 }];
+
+export const todosReducer = (state = initialState, action) => {
+    console.log('Reducer Info', state, action.type, action.payload);
+    switch (action.type) {
+        case ADD_TODO:
+        return Object.assign({}, state, {
```

Comment:
you technically don't need to use Object.assign here as state is an array of objects,  just the spread operator would work with adding the action.payload after.